dessia_common/workflow/core.py

```diff
@@ -1470,7 +1470,8 @@ def dict_to_object(cls, dict_: JsonSerializable, force_generic: bool = False,
         global_dict, pointers_memo = update_pointers_data(global_dict=global_dict, current_dict=dict_,
                                                           pointers_memo=pointers_memo)
 
-        workflow = Workflow.dict_to_object(dict_['workflow'])
+        workflow = Workflow.dict_to_object(dict_=dict_['workflow'], global_dict=global_dict,
```

Comment:
Sadly, I don't have the knowledge to quickly have a look at this modification

Reply:
It's actually a wonder how workflow deserialization could work without these changes. I guess it didn't in some case, and did in some other, but it should never have worked without these new args